shoutem.places/app/screens/PlaceDetails.js

```diff
@@ -167,7 +167,7 @@ export class PlaceDetails extends PureComponent {
       <UniversalLinkButton
         link={place.rsvpLink}
         title={I18n.t('shoutem.cms.rsvpButton')}
-        iconName="rsvp"
+        iconName="Book Online"
```

Comment:
Hi Jan!

this change would not change the text on the button. `iconName` is mañped to the one of the icons from shoutem.ui.
However if you change the title it should be encapsulated with the translations by i18n, but be aware that changing it here would affect all the apps using the shoutem.places extension once they do the update of it. So, my suggestion would be to register your own extension from your fork and than install it to your app. That would allow you to make changes specific to your app at your own pace and would not require CR from anyone.

Reply:
@SoHotSoup Thanks for reply 😄 

@jancoulson Thanks for your submission.
This repository contains code base that is applied to all applications created on Shoutem platform.
Changing anything here would affect all apps, so in here, we keep things as generic as possible.

There is easier & preferred way to change it, with no code required and it is done via Shoutem Builder.
This way, it's changed for your app only.

You can do that by 
- Opening Shoutem builder > Settings > Language & Region
- Under Language tab, you can see your active language. Download the active Language JSON file (check screenshot), open it
- Find string you wish to change (in this case it's the `"rsvpButton": "RSVP" line)`
- Change it, save file and the update Language in builder:
  - Open edit language modal (pencil button)
  - Delete current language file by pressing on it
  - Upload your updated en.json (or any other locale)
  - Hit Save button
  
This way, you can edit any string in app, as per your preferences.


Screenshot showing edit localization & download language json file buttons.

![Screenshot 2024-11-20 at 09 28 08](https://github.com/user-attachments/assets/500d592b-3393-450d-a748-d8f4555c6533)

Reply:
You are welcome, I still feel nostalgic about working at Shoutem.

Reply:
Thanks all. Apologies for going about this the wrong way - it was this page that sent me down the rabbit hole :o). Feel free to close this request. 
![image](https://github.com/user-attachments/assets/8953c0e2-b157-4ef0-a099-bfa7aa0a6fd5)

Reply:
> Thanks all. Apologies for going about this the wrong way - it was this page that sent me down the rabbit hole :o). Feel free to close this request. ![image](https://private-user-images.githubusercontent.com/120024116/388061001-8953c0e2-b157-4ef0-a099-bfa7aa0a6fd5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIxOTkxMjYsIm5iZiI6MTczMjE5ODgyNiwicGF0aCI6Ii8xMjAwMjQxMTYvMzg4MDYxMDAxLTg5NTNjMGUyLWIxNTctNGVmMC1hMDk5LWJmYTdhYTBhNmZkNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyMVQxNDIwMjZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zZmM3NTY3OTU1ZDBiNzExNjgxZjc5NTg5ODJhMzgzOTkxNmY3NGNmNjY3MDEwZTE5ZTMyMGEzMmFlZjI4M2Q1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.07CmGtEXOKZjHQfqxp424uSOJ4Eyz_2jiRHhikO030U)

Hey, hope you've managed to change your localization successfully.

I see why you'd think you have to change the source code, we should probably be more implicit with wording on that screen. Thanks for your input!